Creating-Objects.md

```diff
@@ -9,16 +9,16 @@
 
 ## Introduction
 
-When collecting user input, we often define the kind of input we expect using [[schemas|Schemas]]. These schemas can describe arbitrarily nested collections of base types. We call these base types _objects_. For example, consider this schema:
+When collecting user input, we often define the kind of input we expect using [[schemas|Schemas]]. These schemas can describe arbitrarily nested collections of simple types. We call these simple types _objects_. For example, consider this schema:
 
 ```json
 {
   "type": "dict",
   "properties": [
     {
-      "name": "version",
+      "name": "length",
       "schema": {
-        "type": "int"
+        "type": "Real"
```

Comment:
I think this is still wrong. Schemas that use an object have type='custom' and an obj_type key. See e,g, state_domain_test.py line 4339 and the schema parsing logic in schema_utils.py

Reply:
fixed